lua/spec/team_participant_spec.lua

```diff
@@ -11,7 +11,6 @@ insulate('Team Participant', function()
 		GoldenTest('team_participant', tostring(TeamParticipantsController.fromTemplate{
 			{
```

Comment:
Is there an easy way to force on of the participants to be expanded/show roster during tests?